pkg/volume/hostpath/host_path_test.go

```diff
@@ -197,7 +197,7 @@ func TestProvisioner(t *testing.T) {
 		t.Errorf("Expected reclaim policy %+v but got %+v", v1.PersistentVolumeReclaimDelete, pv.Spec.PersistentVolumeReclaimPolicy)
 	}
 
-	os.RemoveAll(hostPathCreator.basePath)
+	os.RemoveAll(fmt.Sprintf("/tmp/%s", hostPathCreator.basePath))
```

Comment:
without this change, does the test fail?

Reply:
With or without this change, test  all pass. But it actually doesn't cleanup tempfile without this change.

Reply:
thanks!